### DISCUSSION TITLE
Wallet generator MAC

### DISCUSSION THREAD

**Discussion by KevWnrt** [catchthatrabbit/coreminer#78] [Operating System]
Hey! A friend of mine has installed the appropriate file for the wallet on his Mac. Unfortunately, only one text file opens and no wallet generator.

Does anyone have any idea why this could be and could help me?

**Comment by rastislavcore** [comment]
Probably he has to follow the [installation manual](https://github.com/core-coin/wallet-generator#mac-and-linux) and grant permissions.
